'Abs' function unsupported

Generating code for a model that uses 'Abs' tells me that it is not supported.
src/weblab-cg/weblab_cg/_printer.py:203: in _print_Function
   raise ValueError('Unsupported function: ' + str(name))
E   ValueError: Unsupported function: Abs

earm_noble_model_1990.cellml

--- a/weblab_cg/_chaste_printer.py
+++ b/weblab_cg/_chaste_printer.py
@@ -27,9 +27,6 @@ class ChastePrinter(Printer):
         # Make sure we can output a call to GetIntracellularAreaStimulus
         Printer._custom_functions.add('GetIntracellularAreaStimulus')
 
-    def _print_Abs(self, expr):
-        return 'fabs(' + self._print(expr.args[0]) + ')'
-
     def _print_And(self, expr):
         """ Handles logical And. """
         my_prec = precedence(expr)

--- a/weblab_cg/_printer.py
+++ b/weblab_cg/_printer.py
@@ -107,6 +107,9 @@ class Printer(sympy.printing.printer.Printer):
         else:
             self._derivative_function = derivative_function
 
+    def _print_Abs(self, expr):
+        return self._prefix + 'fabs(' + self._print(expr.args[0]) + ')'
+
     def _bracket(self, expr, parent_precedence):
         """
         Converts ``expr`` to string, and adds parentheses around the result, if

--- a/weblab_cg/tests/test_weblab_printer.py
+++ b/weblab_cg/tests/test_weblab_printer.py
@@ -240,3 +240,6 @@ class TestWebLabPrinter(object):
         e = sp.gamma(x)
         with pytest.raises(ValueError):
             p.doprint(e)
+
+    def test_abs(self, p, x, y):
+        assert p.doprint(sp.Abs(x + y)) == 'math.fabs(x + y)'

